### DISCUSSION TITLE
White flash when using Astro with Parallel Plugin

### DISCUSSION THREAD

**Discussion by Eric-llos** [swup/swup#789] [Help] [answered]
Hello there!

I''m trying to replicate an error i'm facing to with stackblitz. It is a small demo but i'm facing the same issue(?) on my local machine with bigger web.

**Demo: https://stackblitz.com/edit/withastro-astro-pupmsf?file=src%2Flayouts%2FLayout.astro**

I have some doubts:

###  -- Maybe astro/swup or maybe parallel plugin doubts 
-When I click in the link, the page does like a flash(or blink), does anyone have any idea why? 

### --I think this is a bug from astro/swup? 
-Bug(?): If I don't set "accessibility: false" to astro.config.*, i get an error in my console: https://ibb.co/c2KGXSQ

### -- Solved doubts / bugs but left them here if can help anyone: 
-It seems to be adding more than two "main" elements (three, to be precise) when transitioning from old to new page. This creates this effect:
[https://ibb.co/5rCXtm8]

-It is not clear for me (sorry) how works the astro/swup config. Can I have a mix config in astro.config.* and whatever when I initialize "new Swup"? Should I only have astro.config.*? If used only astro.config.*, are all the swup plugins available there? 

_-fix for this two points: I've seen the error was coming from having the config in astro.config.* and also initializing it in my layout part. Maybe it would be a good idea to specify in the docs to only have one config? Or the manual one or the astro/swup one? I didn't understand it at first!_

It's my first post here, any positive feedback is appreciated :)

**Comment by daun** [comment]
Welcome 👋

The first error you're seeing is a recent regression in `@swup/astro` 1.3.1. There's [a fix](https://github.com/swup/astro/pull/20) waiting to be approved and merged. In the meantime, downgrading to 1.3.0 should fix it.

As for the other errors, glad you were able to solve them. There's a section on manual swup setup at the end of the readme. If you feel that's too late or if you'd have preferred a hint in the earlier setup section, feel free to create a PR with suggested changes!

**Reply by Eric-llos** [reply]
Hi @daun !

Many thanks for your feedback :) I think I'll write a PR! 

I didn't be able fix the flash/blink one, keeps happening to me. Do you have any idea?

Thanks!<3

**Comment by daun** [comment, marked as answer]
The white flash seems to be caused by not setting the `theme` option to false. It currently applies both your parallel animation as well as the default fade theme opacity animation. See this [forked version of your sandbox](https://stackblitz.com/edit/withastro-astro-x8mjuf?file=astro.config.mjs) where the flash is no longer happening after setting `theme` to false.
